Collectors/UdpCollector.py

```diff
@@ -272,6 +272,12 @@ def on_message(channel, method, properties, body):
             # Send to message queue
             # Get the address and port
             addr = loaded_json['remote'].rsplit(":", 1)
+
+            # High water mark for the queue
+            if len(message_q.qsize() > config.get('AMQP', 'high_water_mark', fallback=100)):
+                while message_q.qsize() > config.get('AMQP', 'low_water_mark', fallback=10):
+                    logging.error("Sleeping for 1 second to allow the queue to drain")
```

Comment:
Might be nice to see a message count.
```suggestion
                    logging.error("Sleeping for 1 second to allow the queue to drain: {} pending".format(message_q.qsize()))
```